planner/core/common_plans.go

```diff
@@ -649,6 +649,41 @@ type SelectInto struct {
 	IntoOpt    *ast.SelectIntoOption
 }
 
+// ExplainInfoForEncode store explain info for JSON encode
+type ExplainInfoForEncode struct {
+	ID                  string `json:"id"`
+	EstRows             string `json:"estRows"`
+	ActRows             string `json:"actRows,omitempty"`
+	TaskType            string `json:"taskType"`
+	AccessObject        string `json:"accessObject"`
+	ExecuteInfo         string `json:"executeInfo,omitempty"`
+	OperatorInfo        string `json:"operatorInfo"`
+	EstCost             string `json:"estCost,omitempty"`
+	CostFormula         string `json:"costFormula,omitempty"`
+	Idx                 int    `json:"idx"`
+	ChildrenIdx         []int  `json:"childrenIdx"`
+	MemoryInfo          string `json:"memoryInfo,omitempty"`
+	DiskInfo            string `json:"diskInfo,omitempty"`
+	TotalMemoryConsumed string `json:"totalMemoryConsumed,omitempty"`
+}
+
+// JSONSlice store JSONExplainRows
+type JSONSlice []*ExplainInfoForEncode
+
+// ToString convert json to string
+func (j *JSONSlice) ToString() (string, error) {
+	b, err := json.Marshal(*j)
```

Comment:
I think this would make the output be `HTMLEscape`d, which is probably not expected.